infrastructure/smart-contracts/contracts/upgrades/node-license/NodeLicense8.sol

```diff
@@ -817,7 +818,9 @@ contract NodeLicense8 is ERC721EnumerableUpgradeable, AccessControlUpgradeable
         address to,
         uint256 amount
     ) external onlyRole(ADMIN_MINT_ROLE) {
-        revert("Not implemented");
+        _validateMint(amount);
```

Comment:
Should we have some parameter validation here too? No zero address or zero amount?

Reply:
Added requires for to and amount.